livekit-agents/livekit/agents/ipc/proc_pool.py

```diff
@@ -66,6 +66,9 @@ def get_by_job_id(self, job_id: str) -> JobExecutor | None:
             None,
         )
 
+    def get_running_jobs(self) -> list[RunningJobInfo]:
+        return [x.running_job for x in self._executors if x.running_job]
+
     def start(self) -> None:
```

Comment:
You can use https://github.com/livekit/agents/blob/0c9eac91938d21ba4c2f2d16cfe05fb8fa237786/livekit-agents/livekit/agents/worker.py#L303 instead